vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -233,7 +233,8 @@ def _get_scheme_from_parts(
         if is_activation_quantization_format(self.quant_format):
             if self._is_fp8_w8a8(weight_quant, input_quant):
                 is_fp8_w8a8_supported = self._check_scheme_supported(
-                    CompressedTensorsW8A8Fp8.get_min_capability(), error=False)
+                    CompressedTensorsW8A8Fp8.get_min_capability(),
+                    error=False) if torch.cuda.is_available() else True
```

Comment:
How does that affect non cuda, non hpu scenarios?

Reply:
Hi @michalkuligowski 
We couldn't find a way to query for specific feature support. There is no device abstraction for custom ops and determining support based on cuda version is not sufficient. Couldn't find a way to generalize for non cuda devices, including hpu.
I proposed to Konrad to have some custom op abstraction to hide cuda, rocm, tpu, ipex and hpu ops under a common structure similar to what has been done for model_executor/custom_op, but it would require a some refactoring out of scope for this PR.
Until we have a way to do that we could change the code above to return True only if current platform is HPU.
I guess it should be safer for other platforms. Is it acceptable?

Reply:
If code breaks other devices than HPU it looses capability to upstream. We need to keep code able to upstream at all times